script-library/docs/gradle.md

```diff
@@ -1,3 +1,9 @@
+**IMPORTANT NOTE: We're starting to migrate contents of this repo to the [devcontainers org](https://github.com/devcontainers), as part of the work on the [open dev container specification](https://containers.dev).**
+
+**We've currently migrated `gradle` as part of the `java` Feature in [here](https://github.com/devcontainers/features/tree/main/src/java#options).**
```

Comment:
Rather than in here (which may sound a bit mysterious - what is "here" 😄), maybe:

```suggestion
**We've currently migrated `gradle` as [part of the `java` Feature](https://github.com/devcontainers/features/tree/main/src/java#options).**
```